types/bitcoinjs-lib/index.d.ts

```diff
@@ -232,172 +352,969 @@ export const opcodes: {
     OP_WITHIN: number;
     OP_XOR: number;
 };
+
 export namespace HDNode {
     namespace MASTER_SECRET {
```

Comment:
Looks like `MASTER_SECRET` should be a static field of `HDNode` with the type `Buffer`. Then this whole namespace can be removed.